<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>myInsertAsset</class>
 <widget class="QDialog" name="myInsertAsset">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>179</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加资产</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>140</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelAccount">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>371</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,3">
      <item>
       <layout class="QHBoxLayout" name="hlMarket">
        <item>
         <widget class="QRadioButton" name="radioSH">
          <property name="text">
           <string>上海</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioSZ">
          <property name="text">
           <string>深圳</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioOther">
          <property name="text">
           <string>其它</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="labelAssetCode">
          <property name="text">
           <string>资产代号：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEditAssetCode"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelAssetName">
          <property name="text">
           <string>资产名称：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEditAssetName"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="2" column="0">
       <widget class="QLabel" name="labelPrice">
        <property name="text">
         <string>单价：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelType">
        <property name="text">
         <string>类别：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEditType"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelAmount">
        <property name="text">
         <string>数量：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="spinBoxPrice">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>-1000000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinBoxAmount">
        <property name="maximum">
         <number>1000000</number>
        </property>
        <property name="singleStep">
         <number>100</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditAssetCode</tabstop>
  <tabstop>spinBoxAmount</tabstop>
  <tabstop>spinBoxPrice</tabstop>
  <tabstop>lineEditType</tabstop>
  <tabstop>lineEditAssetName</tabstop>
  <tabstop>radioSH</tabstop>
  <tabstop>radioSZ</tabstop>
  <tabstop>radioOther</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>myInsertAsset</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>myInsertAsset</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
